src/LogModel/PublicAPI.Shipped.txt

```diff
@@ -0,0 +1 @@
+#nullable enable
```

Comment:
Is this file needed?

Reply:
It looks like if I remove it, Roslyn code analysis will give me build warnings saying it can't find this file